source/utilities.tex

```diff
@@ -1249,10 +1249,10 @@
 \begin{itemdescr}
 \pnum
 \returns \tcode{pair<V1, V2>(std::forward<T1>(x), std::forward<T2>(y))},
-where \tcode{V1} and \tcode{V2} are determined as follows: Let \tcode{Ui} be
-\tcode{decay_t<Ti>} for each \tcode{Ti}. If \tcode{Ui} is a specialization
-of \tcode{reference_wrapper}, then \tcode{Vi} is \tcode{Ui::type\&},
-otherwise \tcode{Vi} is \tcode{Ui}.
+where \tcode{V1} and \tcode{V2} are determined as follows: Let $\tcode{U}_i$ be
+\tcode{decay_t<$\tcode{T}_i$>} for each $\tcode{T}_i$. If $\tcode{U}_i$ is a specialization
```

Comment:
I'm not sure about this change. I have considered this section myself, and I deliberately didn't want to call the same type both `T1` and `T_1`. We should consistently stick with one name (which is `T1` in this case). It's OK to define *new* types `U_i` as the elements of a pack, say, but unless you say "Let T_1 be T1 and T_2 be T2", I'm concerned that this is making things less clear.